Homework_9_Employees_New_Version/New_wersionHW9_Employees/src/com/exemplee/mployees/newversion/Employee.java

```diff
@@ -0,0 +1,64 @@
+package com.exemplee.mployees.newversion;
+
+public class Employee {
+
+
+    private String name;
+    private int age;
+    private int salary;
+
+    public Employee(String name, int age, int salary) {
+        this.name = name;
+        this.age = age;
+        this.salary = salary;
+
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public void setName(String name) {
+        this.name = name;
+    }
+
+    public int getAge() {
+        return age;
+    }
+
+    public void setAge(int age) {
+        this.age = age;
+    }
+
+    public int getSalary() {
+        return salary;
+    }
+
+    public void setSalary(int salary) {
+        this.salary = salary;
+    }
+
+
+    public void work () {
+            System.out.println(name + " is working.");
+        }
+        public void takeBreak () {
```

Comment:
Please don't forget to apply proper code formatting (indentations)

`Ctrl + Alt + L` in IDEA would help